scripts/scil_filter_tractogram_anatomically.py

```diff
@@ -69,6 +69,17 @@
                              assert_inputs_exist,
                              assert_output_dirs_exist_and_empty)
 
+EPILOG = """
+    References:
+        [1] Jörgens, D., Descoteaux, M., Moreno, R., 2021. Challenges for
+        tractogram ﬁltering. In: Özarslan, E., Schultz, T., Zhang, E., Fuster,
+        A. (Eds.), Anisotropy Across Fields and Scales. Springer. Mathematics
+        and Visualization.
+        [2] Legarreta, J., Petit, L., Rheault, F., Theaud, G., Lemaire, C.,
+        Descoteaux, M., Jodoin, P.M. Filtering in tractography using
+        autoencoders (FINTA). Medical Image Analysis. 2021
+    """
+
 
 def _build_arg_parser():
     p = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
```

Comment:
You need to add the EPILOG here. Check scil_compute_pft.py to know how.

Reply:
Since it's quite a long reference I did it like in scil_compute_todi with epilog=EPILOG. Is that ok or you want it exactly like that?